### DISCUSSION TITLE
SOPS is unmaintained

### DISCUSSION THREAD

**Discussion by jr0dd** [fluxcd/flux2#2270] [Proposals]
Since Mozilla is no longer maintaining SOPS, I'm curious how the fluxcd devs would feel about forking it. Just tossing the idea out there since it is such in invaluable companion for Flux for many people. It easily allows people to share their code in a public repo, which in turn also gives other people ideas and solutions to their issues. 

Here is the current SOPS discussion regarding their lack of time/interest in the state of the project https://github.com/mozilla/sops/discussions/927

**Comment by onedr0p** [comment]
SOPS being unmaintained is kind of a bummer, and I hope they get some people on to maintain it. The current maintainers don't even have time to respond to the issue they created about asking for help 😞

**Reply by monadic** [reply]
hi @jr0dd please could you contact me alexis at weave dot works

**Reply by onedr0p** [reply]
@monadic He's just notifying the Flux developers and the community here about mozilla/sops being left unmaintained.

**Comment by stefanprodan** [comment]
> SOPS is such in invaluable companion for Flux for many people

I agree that SOPS is a great alternative to Sealed Secrets or any other controller that manages secrets using custom resources. But the Flux team has no resources to take over SOPS, we can hardly keep up with Flux feature requests.

**Comment by nickperkins** [comment]
Apparently Mozilla are taking back the wheel https://github.com/mozilla/sops/discussions/927#discussioncomment-2183834
